docs/reference/modules/discovery/bootstrapping.asciidoc

```diff
@@ -0,0 +1,110 @@
+[[modules-discovery-bootstrap-cluster]]
+=== Bootstrapping a cluster
+
+Starting an Elasticsearch cluster for the very first time requires the initial
+set of master-eligible nodes to be explicitly set on one or more of the
+master-eligible nodes in the cluster. This is known as _cluster bootstrapping_.
+This is only required the very first time the cluster starts up: nodes that
+have already joined a cluster will store this information in their data folder,
+and freshly-started nodes that are intended to join an existing cluster will
+obtain this information from the cluster's elected master. This information is
+given using this setting:
+
+`cluster.initial_master_nodes`::
+
+    Sets a list of the <<node.name,node names>> or transport addresses of the
+    initial set of master-eligible nodes in a brand-new cluster. By default
+    this list is empty, meaning that this node expects to join a cluster that
+    has already been bootstrapped.
+
+This setting can be given on the command line when starting up each
+master-eligible node, or added to the `elasticsearch.yml` configuration file on
```

Comment:
This paragraph makes it sound like you'd have to do this on every master-eligible node, whereas the initial paragraph made it sound like you only had to do it on one of them.